devops/docker/Dockerfile.linting

```diff
@@ -1,7 +1,8 @@
-FROM circleci/python:2
+FROM circleci/python:2-stretch
```

Comment:
For linting, could we use an alpine based image to cut down on image sizes?

Reply:
I'd rather not because it would impose that people know about Alpine in terms of packaging, dependencies etc. We have to know about stretch already because it's what Tails is made from. If you feel strongly about saving on image size versus reducing the amount of knowledge necessary to maintain CI, I'd be happy to discuss that in a thread in the forum.

Reply:
Ok, so this isn't a blocker for this but I'll post to the forum some time for that.